pkgs/by-name/cl/clevis/package.nix

```diff
@@ -1,6 +1,6 @@
 {
   lib,
-  asciidoc,
+  asciidoc-full,
```

Comment:
This can be squashed to the `by-name` transition commit IMO.

Reply:
The `by-name` migration should not change the API. Ideally it should be as simple as renaming the files and deleting the reference on all-packages.nix

This way we can have some confidence on future bisections.